PendingReleaseNotes

```diff
@@ -14,6 +14,13 @@
   If you want to return back the old behavior add 'ssl_options=' (empty) to
   ``rgw frontends`` configuration.
 
+* MGR: The pg_autoscaler has a new 'scale-down' profile which provides more
+  performance from the start for new pools. The default remains the existing
+  behavior, now called the 'scale-up' profile, since changing profiles causes
+  data movement. For more details, see:
+
+  doc/rados/operations/placement-groups.rst
```

Comment:
```suggestion
  https://docs.ceph.com/en/latest/rados/operations/placement-groups/
```

Reply:
On my vstart cluster, running
 ``` 
ceph osd pool set autoscale-profile scale-down 
``` 
results in  

``` 
autoscale-profile is already a scale-down! 
```
which does not match the description above, so something is not right!